File: backend/src/main/java/com/atl/bakuboutiquehub/dto/ProductDTO.java
package com.atl.bakuboutiquehub.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ProductDTO {
    private Long id;
    private String name;
    private String description;
    private BigDecimal price;
    private String imageUrl;
    private Integer stockCount;
    private Long boutiqueId;
}

File: app/src/main/java/com/atl/bakuboutiquehub/network/model/ProductDTO.kt
package com.atl.bakuboutiquehub.network.model

data class ProductDTO(
    val id: Long? = null,
    val name: String,
    val description: String? = null,
    val price: Double,
    val imageUrl: String? = null,
    val stockCount: Int? = null,
    val boutiqueId: Long? = null
)
